delivery_carrier_package_measure_required/models/stock_picking.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _check_required_package_measurement(self):
+        self.ensure_one()
+        if not self.carrier_id:
```

Comment:
This needs to use `ship_carrier_id` from `stock_picking_delivery_link` module as the carrier is defined only on the ship step and here we likely are on a pack step in the chain of pick-pack-ship

Reply:
@jbaudoux If we move the required measurement flag on to the `product.packaging` model there is no need to check anything on the carrier. 
Would that be coherent for you ?

Reply:
Yes, that's correct. It's easier. Only the delivery package will tell what you need as measurement.

Reply:
Done